### ISSUE TITLE
Focus outline missing from 2D and 3D editor viewports

### ISSUE THREAD

**Issue by Giganzo** [godotengine/godot#100170]
### Tested versions

v4.4.dev6.official [1f47e4c4e]

### System information

Fedora Linux 40 (KDE Plasma) on Wayland - X11 display driver, Multi-window

### Issue description

The focus outline when moving mouse over the editor viewport is missing.

4.4-dev5:

https://github.com/user-attachments/assets/b0221361-29ec-488a-9a5e-18f8336868de

4.4-dev6:

https://github.com/user-attachments/assets/c035942d-1270-4a42-a925-823e8d398a28

_video is using v4.4.dev.custom_build [aa8d9b83f]_

### Steps to reproduce

Set focus to anything (like scene tree) and move mouse over the editor viewport

### Minimal reproduction project (MRP)

N/A
